plugins/InspectionGadgets/src/com/siyeh/ig/redundancy/RedundantStringOperationInspection.java

```diff
@@ -565,7 +646,78 @@ private static String getTargetString(@NotNull final PsiMethodCallExpression cal
         );
       }
     }
+  }
+
 
+  private static class RemoveRedundantChangeCaseFix implements LocalQuickFix {
+    private final @NotNull String caseRedundant;
+    private final @NotNull PlaceCaseEqualType myPlaceCaseEqualType;
+    private final static String equalsIgnoreCase = "equalsIgnoreCase";
```

Comment:
There's `com.siyeh.HardcodedMethodConstants#EQUALS_IGNORE_CASE`. Though it's also quite fine to use method names inline, without extracting them to constants.

Reply:
Thank you. I changed to com.siyeh.HardcodedMethodConstants#EQUALS_IGNORE_CASE.

Thank you again for checking!